frontend/lib/src/components/graph.dart

```diff
@@ -59,30 +58,33 @@ class Graph extends StatelessWidget {
                   height: 150,
                   child: StatefulBuilder(
                     builder: (context, stateful) {
-                      return PieChart(
-                        PieChartData(
-                          pieTouchData: PieTouchData(
-                            touchCallback:
-                                (FlTouchEvent event, pieTouchResponse) {
-                              stateful(() {
-                                if (!event.isInterestedForInteractions ||
-                                    pieTouchResponse == null ||
-                                    pieTouchResponse.touchedSection == null) {
-                                  graphController.setFocusIndex = -1;
+                      return Obx(
+                        () => PieChart(
+                          PieChartData(
+                            pieTouchData: PieTouchData(
+                              touchCallback:
+                                  (FlTouchEvent event, pieTouchResponse) {
+                                stateful(() {
+                                  if (!event.isInterestedForInteractions ||
+                                      pieTouchResponse == null ||
+                                      pieTouchResponse.touchedSection == null) {
+                                    graphController.setFocusIndex = -1;
 
-                                  return;
-                                }
-                                graphController.setFocusIndex = pieTouchResponse
-                                    .touchedSection!.touchedSectionIndex;
-                              });
-                            },
+                                    return;
+                                  }
+                                  graphController.setFocusIndex =
+                                      pieTouchResponse
+                                          .touchedSection!.touchedSectionIndex;
+                                });
+                              },
+                            ),
+                            borderData: FlBorderData(
+                              show: false,
+                            ),
+                            sectionsSpace: 0,
+                            centerSpaceRadius: 30,
+                            sections: showingSections(),
```

Comment:
```suggestion
                            sections: showingSections(controller.list),
```
pass here list so that this can be updated when the observable updates